starlite/testing/async_test_client/life_span_handler.py

```diff
@@ -0,0 +1,12 @@
+from typing import TYPE_CHECKING
+
+from starlite.testing.base.life_span_handler_base import BaseLifeSpanHandler
+
+if TYPE_CHECKING:
+    from starlite.testing import AsyncTestClient
+    from starlite.types import LifeSpanReceiveMessage  # noqa: F401  # nopycln: import
+
+
+class LifeSpanHandler(BaseLifeSpanHandler):
```

Comment:
this is redundant - you can simply create a single lifespan handler that receives a generic for the client argument, then you dont need to subclass this.